docs/user/run.md

```diff
@@ -34,6 +34,7 @@ hostname (`-h $HOSTNAME`) and using the `-ip` Registrator option below.
 Option                           | Since | Description
 ------                           | ----- | -----------
 `-internal`                      |       | Use exposed ports instead of published ports
+`appendInternalPort`             |       | Append internal service port in services with only one port exposed
```

Comment:
If you have `./registrator --appendInternalPort true ....` and your service `docker run -p 80:80 -p 443:443 -e SERVICE_80_NAME=foo-http -e SERVICE_443_NAME=foo-https nginx` your backend will be named `foo-http` and `foo-https` instead `foo-http-80` and `foo-https-443` respectively.
https://github.com/ch3lo/registrator/blob/da3cd457a39c0d280104175ca42c5791fb03fd1b/bridge/bridge.go#L250-L253